From d505707379d64e6e1a4ade23db3547ffc5a050a6 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Sun, 25 Dec 2016 16:09:44 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399: Rename OPP nodes as
 opp@<opp-hz>

It would be better to name OPP nodes as opp@<opp-hz> as that will ensure
that multiple DT nodes don't contain the same frequency. Of course we
expect the writer to name the node with its opp-hz frequency and not any
other frequency.

And that will let the compile error out if multiple nodes are using the
same opp-hz frequency.

Change-Id: I8c77646329e39390fb135d4d75d34893a8168876
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 .../boot/dts/rockchip/rk3399-early-opp.dtsi   | 28 +++++------
 arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi  | 50 +++++++++----------
 2 files changed, 39 insertions(+), 39 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-early-opp.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-early-opp.dtsi
index c6ba6e1a477b..ac848c3a00a9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-early-opp.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-early-opp.dtsi
@@ -54,28 +54,28 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <900000>;
 			clock-latency-ns = <40000>;
 		};
-		opp01 {
+		opp@600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <900000>;
 		};
-		opp02 {
+		opp@816000000 {
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <900000>;
 		};
-		opp03 {
+		opp@1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
 			opp-microvolt = <900000>;
 		};
-		opp04 {
+		opp@1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
 			opp-microvolt = <950000>;
 		};
-		opp05 {
+		opp@1416000000 {
 			opp-hz = /bits/ 64 <1416000000>;
 			opp-microvolt = <1050000>;
 		};
@@ -85,24 +85,24 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <900000>;
 			clock-latency-ns = <40000>;
 		};
-		opp01 {
+		opp@600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <900000>;
 		};
-		opp02 {
+		opp@816000000 {
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <900000>;
 		};
-		opp03 {
+		opp@1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
 			opp-microvolt = <950000>;
 		};
-		opp04 {
+		opp@1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
 			opp-microvolt = <100000>;
 		};
@@ -111,15 +111,15 @@
 	gpu_opp_table: opp-table2 {
 		compatible = "operating-points-v2";
 
-		opp00 {
+		opp@200000000 {
 			opp-hz = /bits/ 64 <200000000>;
 			opp-microvolt = <900000>;
 		};
-		opp01 {
+		opp@297000000 {
 			opp-hz = /bits/ 64 <297000000>;
 			opp-microvolt = <900000>;
 		};
-		opp02 {
+		opp@400000000 {
 			opp-hz = /bits/ 64 <400000000>;
 			opp-microvolt = <900000>;
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi
index 5f7bea6600bd..d3a5171b121d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-opp.dtsi
@@ -47,33 +47,33 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <800000>;
 			clock-latency-ns = <40000>;
 		};
-		opp01 {
+		opp@600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <800000>;
 			clock-latency-ns = <40000>;
 		};
-		opp02 {
+		opp@816000000 {
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <850000>;
 			clock-latency-ns = <40000>;
 			opp-suspend;
 		};
-		opp03 {
+		opp@1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
 			opp-microvolt = <925000>;
 			clock-latency-ns = <40000>;
 		};
-		opp04 {
+		opp@1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
 			opp-microvolt = <1000000>;
 			clock-latency-ns = <40000>;
 		};
-		opp05 {
+		opp@1416000000 {
 			opp-hz = /bits/ 64 <1416000000>;
 			opp-microvolt = <1125000>;
 			clock-latency-ns = <40000>;
@@ -84,43 +84,43 @@
 		compatible = "operating-points-v2";
 		opp-shared;
 
-		opp00 {
+		opp@408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <800000>;
 			clock-latency-ns = <40000>;
 		};
-		opp01 {
+		opp@600000000 {
 			opp-hz = /bits/ 64 <600000000>;
 			opp-microvolt = <800000>;
 			clock-latency-ns = <40000>;
 		};
-		opp02 {
+		opp@816000000 {
 			opp-hz = /bits/ 64 <816000000>;
 			opp-microvolt = <825000>;
 			clock-latency-ns = <40000>;
 			opp-suspend;
 		};
-		opp03 {
+		opp@1008000000 {
 			opp-hz = /bits/ 64 <1008000000>;
 			opp-microvolt = <875000>;
 			clock-latency-ns = <40000>;
 		};
-		opp04 {
+		opp@1200000000 {
 			opp-hz = /bits/ 64 <1200000000>;
 			opp-microvolt = <950000>;
 			clock-latency-ns = <40000>;
 		};
-		opp05 {
+		opp@1416000000 {
 			opp-hz = /bits/ 64 <1416000000>;
 			opp-microvolt = <1025000>;
 			clock-latency-ns = <40000>;
 		};
-		opp06 {
+		opp@1608000000 {
 			opp-hz = /bits/ 64 <1608000000>;
 			opp-microvolt = <1100000>;
 			clock-latency-ns = <40000>;
 		};
-		opp07 {
+		opp@1800000000 {
 			opp-hz = /bits/ 64 <1800000000>;
 			opp-microvolt = <1200000>;
 			clock-latency-ns = <40000>;
@@ -130,27 +130,27 @@
 	gpu_opp_table: opp-table2 {
 		compatible = "operating-points-v2";
 
-		opp00 {
+		opp@200000000 {
 			opp-hz = /bits/ 64 <200000000>;
 			opp-microvolt = <800000>;
 		};
-		opp01 {
+		opp@297000000 {
 			opp-hz = /bits/ 64 <297000000>;
 			opp-microvolt = <800000>;
 		};
-		opp02 {
+		opp@400000000 {
 			opp-hz = /bits/ 64 <400000000>;
 			opp-microvolt = <825000>;
 		};
-		opp03 {
+		opp@500000000 {
 			opp-hz = /bits/ 64 <500000000>;
 			opp-microvolt = <875000>;
 		};
-		opp04 {
+		opp@594000000 {
 			opp-hz = /bits/ 64 <594000000>;
 			opp-microvolt = <925000>;
 		};
-		opp05 {
+		opp@800000000 {
 			opp-hz = /bits/ 64 <800000000>;
 			opp-microvolt = <1100000>;
 		};
@@ -159,23 +159,23 @@
 	dmc_opp_table: opp-table3 {
 		compatible = "operating-points-v2";
 
-		opp00 {
+		opp@200000000 {
 			opp-hz = /bits/ 64 <200000000>;
 			opp-microvolt = <825000>;
 		};
-		opp01 {
+		opp@297000000 {
 			opp-hz = /bits/ 64 <297000000>;
 			opp-microvolt = <850000>;
 		};
-		opp02 {
+		opp@400000000 {
 			opp-hz = /bits/ 64 <400000000>;
 			opp-microvolt = <850000>;
 		};
-		opp03 {
+		opp@594000000 {
 			opp-hz = /bits/ 64 <594000000>;
 			opp-microvolt = <900000>;
 		};
-		opp04 {
+		opp@800000000 {
 			opp-hz = /bits/ 64 <800000000>;
 			opp-microvolt = <900000>;
 		};
-- 
2.35.3

